app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -141,6 +141,31 @@ def bulk_cancel
         }
       end
 
+      api :POST, '/tasks/bulk_stop', N_('Stop all stoppable tasks')
+      param :search, String, :desc => N_('Stop tasks matching search string')
+      param :task_ids, Array, :desc => N_('Stop specific tasks by ID')
```

Comment:
Probably out of scope of this PR, but how about we drop the `task_ids` param everywhere. If someone wants to target tasks by id, they would use search with something like `id ^ (1, 2, 3)`?

Reply:
Yeah that makes sense, added: https://projects.theforeman.org/issues/29913